pkg/orchestrator/orchestrator.go

```diff
@@ -198,7 +198,11 @@ func (s *Orchestrator) TaskExists(ctx context.Context, taskType backend.TaskType
 
 // Taking tasks defined in the config, look up their ID (e.g. ARN) in the given Stack
 // object, and run these tasks with TaskRunner
-func (s *Orchestrator) RunTasks(ctx context.Context, stack *stack_mgr.Stack, taskType backend.TaskType) error {
+func (s *Orchestrator) RunTasks(ctx context.Context, stack *stack_mgr.Stack, taskType backend.TaskType, dryRun bool) error {
+	if dryRun {
+		return nil
+	}
+
```

Comment:
```suggestion
func (s *Orchestrator) RunTasks(ctx context.Context, stack *stack_mgr.Stack, taskType backend.TaskType) error {
	if s.dryRun {
		return nil
	}

```

I would make the dryRun bool a property of Orchestrator. Then add:

~~~go

func (s *Orchestrator) WithDryRun(dryRun bool) *Orchestrator {
	s.dryRun = dryRun
	return s
}
~~~

Reply:
Done.